docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -66,7 +66,9 @@ Install Einstein for Developers:
 
 - If you are using Code Builder, click the Extensions icon in the Activity Bar in Code Builder, search for “Einstein for Developers” and click **Install**.
 
-**Note**: To use Einstein for Developers locally, you must have the [Salesforce Extensions pack]() installed in your VS Code for desktop application. See [Install Salesforce Extensions](https://developer.salesforce.com/tools/vscode/en/vscode-desktop/install) for more information.
+- If you have any other "competing" AI-enabled extensions such as GitHub Copilot installed, click the Extensions icon in the Activity Bar, search for the extension by name and click **Disable** in the extension's marketplace page. Do this for all installed AI-enabled extensions.
```

Comment:
What do you think about something like...

If you have multiple AI-enabled extensions installed in VS Code, you may run into some weird behavior. We recommend you disable all other AI-enabled extensions when working with Einstein for Developers. Follow the steps below:
1. Click the Extensions icon in the Activity Bar
2. Search for the extension by name
3. Click **Disable** in the extension's marketplace page.

Repeat for all other AI-based tooling extensions and you'll be ready to go with Einstein for Developers!

Reply:
Ok. I can work with your suggestion. We don't want to say "weird behavior". Can we be more specific? Is it "incorrect code completions"?

Reply:
upated per your comments, @AnanyaJha